templates/account/signup.html

```diff
@@ -85,4 +105,41 @@ <h2 class="text-xl text-center">Or Sign Up with Email</h2>
 
   </div>
 
+  <script>
+
+    function toggleButtonsOff(state) {
+      let gitButton = document.getElementById("gitButton");
+      let googleButton = document.getElementById("googleButton");
+      let signupButton = document.getElementById("signup");
+      if (state) {
+        gitButton.classList.add("disabledLink");
+        googleButton.classList.add("disabledLink");
+        signupButton.classList.add("disabledLink");
+        return
+      }
+        gitButton.classList.remove("disabledLink");
+        googleButton.classList.remove("disabledLink");
+        signupButton.classList.remove("disabledLink");
+      return
+    }
```

Comment:
**suggestion:** How about we use an array of button ids and `forEach` (or for loop)  to handle multiple buttons and reduce repetition. Makes it easier for us to add/remove in the future
**todo:** Remove the `return` statements since they are unnecessary for void functions

Reply:
This is great, will implement!